contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.4.0--3.5.0.sql

```diff
@@ -2624,6 +2943,8 @@ CALL sys.babelfish_drop_deprecated_object('view', 'sys', 'check_constraints_depr
 CALL sys.babelfish_drop_deprecated_object('view', 'sys', 'types_deprecated_3_5_0');
 CALL sys.babelfish_drop_deprecated_object('view', 'sys', 'table_types_deprecated_3_5_0');
 CALL sys.babelfish_drop_deprecated_object('view', 'sys', 'systypes_deprecated_3_5_0');
+CALL sys.babelfish_drop_deprecated_object('view', 'information_schema_tsql', 'columns');
+CALL sys.babelfish_drop_deprecated_object('view', 'sys', 'spt_columns_view_managed_3_5_0');
```

Comment:
We changes the sys.objects object_id column to INT somewhere down the line but the corresponding drop & recreate for sys.spt_columns_view_managed is missing. So after upgrade from source = PG13 we still have spt_columns_view_managed.object_id as an OID datatype and upgrade failures.
Hence, need to drop and re create this now.